CodeStyleExercise/ColorsViewController.swift

```diff
@@ -8,88 +8,62 @@
 
 import UIKit
 
-class ColorsViewController : UIViewController, UICollectionViewDataSource, UITextFieldDelegate
-{
-
-    class constants {
-        static let DFLTNUMCOLORS:Int=12
-    }
-
-    @IBOutlet weak var collectionView:UICollectionView!
-    @IBOutlet weak var colorCountTextField: UITextField!
-
+final class ColorsViewController: UIViewController, UICollectionViewDataSource, UITextFieldDelegate {
+    
+    @IBOutlet private weak var collectionView: UICollectionView!
+    @IBOutlet private weak var colorCountTextField: UITextField!
+    
     var ct: Int? = 12
-
+    
+    class Constants {
+        static let defaultColorNumber = 12
+    }
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.collectionView.register(UINib(nibName: "ColorsCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "Cell")
-
-        self.colorCountTextField.placeholder = "12"
+        collectionView.register(UINib(nibName: "ColorsCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "Cell")
+        colorCountTextField.text = "\(Constants.defaultColorNumber)"
     }
-
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-    }
-
-    // MARK: - Collection View Data Source Methods
-
-    func collectionView(_ collectionView:UICollectionView, numberOfItemsInSection section : Int) -> Int{
+    
+    // MARK: - CollectionViewDataSource
+    func collectionView(_ collectionView:UICollectionView, numberOfItemsInSection section : Int) -> Int {
         return ct!
```

Comment:
By force-unwrapping this value, the app crashes if the user leaves the "How Many?" field blank. See the solution code for better handling.